docker-images/scripts/admin-client.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-java -jar /admin.jar "$@"
\ No newline at end of file
+export CLASSPATH="/admin.jar:${CLASSPATH}"
+
+java -cp $CLASSPATH io.strimzi.Main
```

Comment:
```suggestion
java -cp $CLASSPATH io.strimzi.Main "$@"
```
in case that we will not keep it there, we would not be able to execute the sub-commands
for example `admin-client topic ...`